app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    user.update(posts_counter: user.posts.count)
+  end
```

Comment:
- [ ] Also there, inside of using `user`, It could be `author`, let's take an example from this: 

```ruby
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes

after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```

Reply:
Completed as per instructions